server-nodejs/routes/cruds.js

```diff
@@ -64,6 +58,11 @@ module.exports = function (app, express) {
         var author = req.user.username || req.connection.remoteAddress;
         var time = Date.now();
         for (var n in nodes) {
+            if ('object' !== typeof nodes[n]) {
+                res.status(400);
+                res.send('create error: the body of the request is empty');
```

Comment:
i can't merge this commit because I don't understand this change in the create route. a commit to improve the tests should just improve the tests not implement a new behavior "made tests use right Content-Type"